src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -234,8 +234,9 @@ private void AddComponentStats(StringBuilder builder)
             {
                 var chainHeadersBehavior = peer.Behavior<ConsensusManagerBehavior>();
 
-                string peerHeights = $"(r/s):{(chainHeadersBehavior.BestReceivedTip != null ? chainHeadersBehavior.BestReceivedTip.Height.ToString() : peer.PeerVersion?.StartHeight + "*" ?? "-")}";
+                string peerHeights = $"(r/s/c):{(chainHeadersBehavior.BestReceivedTip != null ? chainHeadersBehavior.BestReceivedTip.Height.ToString() : peer.PeerVersion?.StartHeight + "*" ?? "-")}";
                 peerHeights += $"/{(chainHeadersBehavior.BestSentHeader != null ? chainHeadersBehavior.BestSentHeader.Height.ToString() : peer.PeerVersion?.StartHeight + "*" ?? "-")}";
+                peerHeights += $"/{chainHeadersBehavior.GetCachedItemsCount()}";
```

Comment:
instead of 
```
a = "q"
a += "w"
```

better to do 

```
a = "q" +
      "w"
```

mostly for consistency

Reply:
consistency with what?
I saw the code was already doing a += (L238) and so I used that syntax for consistency too
It's ok for me to fix it if it break the way we are used to handle string composition (even if tbh I'd use a StringBuilder to do those concatenations)

Reply:
small change to make, Fixed.

Reply:
>consistency with what?

with the majority of the codebase. 
but it's up to you to fix it or not, it's not a policy or smth